### PR TITLE
GL-204 Adds Swipe tooltip to media gallery on mobile

### PR DESCRIPTION
**Pull request by milindmore22**

### CODE REVIEW

**Comment by abhijitrakas** [comment]
I think we need to add message below image like 'Swipe for more media'. Also I found that this swipe image is the same image which is suggested by client. It will be better if we use different image which should be user friendly. @milindmore22

**Comment by abhijitrakas** [comment]
Also we need to remove swipe popup on swipe as well. Currently if we click on popup then only it is remove. @milindmore22

**Comment by abhijitrakas** [comment]
I think we need to add message below image like 'Swipe for more media'. Also I found that this swipe image is the same image which is suggested by client. It will be better if we use different image which should be user friendly. @milindmore22

**Comment by milindmore22** [comment]
@abhijitrakas please suggest swipe icon
https://thenounproject.com/term/swipe/

**Comment by milindmore22** [comment]
@abhijitrakas Addressed feedback please recheck.

File: app/assets/js/rtMedia.js [hunk 1]
```diff
@@ -547,6 +547,33 @@ jQuery( 'document' ).ready( function( $ ) {
...
+		    // On touch hide tooltip.
+		    jQuery( '#mobile-swipe-overlay' ).on ( 'click', function( e ) {
+			setCookie( 'rtmedia-touch-swipe-tooltip' , true, 365 );
+			jQuery( this ).hide();
+		    } );
+		    
+		    // On swipe hide tooltip.
+		    jQuery( '#mobile-swipe-overlay' ).swipe( {
+			 //Generic swipe handler for all directions
+			swipe:function(event, direction, distance, duration, fingerCount, fingerData) {
```
Comment (thrijith) [inline app/assets/js/rtMedia.js]: space before and after `(`

could be `swipe:function( event, direction, distance, duration, fingerCount, fingerData )`

File: app/assets/js/rtMedia.js [hunk 2]
```diff
@@ -568,6 +595,41 @@ jQuery( 'document' ).ready( function( $ ) {
...
 
+	/**
+	 * Sets Cookie.
+	 * 
+	 * @param {string} cname
+	 * @param {string} cvalue
+	 * @param {int} exdays
+	 * @return void
+	 */
+	function setCookie(cname, cvalue, exdays) {
```
Comment (thrijith) [inline app/assets/js/rtMedia.js]: could be `setCookie( cname, cvalue, exdays )`

File: app/assets/js/rtMedia.js [hunk 3]
```diff
@@ -568,6 +595,41 @@ jQuery( 'document' ).ready( function( $ ) {
...
+	    var d = new Date();
+	    d.setTime(d.getTime() + (exdays * 24 * 60 * 60 * 1000));
+	    var expires = "expires="+d.toUTCString();
```
Comment (thrijith) [inline app/assets/js/rtMedia.js]: could be
`d.setTime( d.getTime() + ( exdays * 24 * 60 * 60 * 1000 ) );`

`var expires = "expires=" + d.toUTCString();`

File: app/assets/js/rtMedia.js [hunk 4]
```diff
@@ -568,6 +595,41 @@ jQuery( 'document' ).ready( function( $ ) {
...
+	    document.cookie = cname + "=" + cvalue + ";" + expires + ";path=/";
+	}
+
+	/**
+	 * Get Cookie.
+	 * 
+	 * @param {string} cname
+	 * @return {string}
+	 */
+	function getCookie(cname) {
```
Comment (thrijith) [inline app/assets/js/rtMedia.js]: `getCookie( cname )`

File: app/assets/js/rtMedia.js [hunk 5]
```diff
@@ -568,6 +595,41 @@ jQuery( 'document' ).ready( function( $ ) {
...
+		var c = ca[i];
+		while (c.charAt(0) == ' ') {
+		    c = c.substring(1);
+		}
+		if (c.indexOf(name) == 0) {
+		    return c.substring(name.length, c.length);
+		}
+	    }
+	    return "";
+	}
```
Comment (thrijith) [inline app/assets/js/rtMedia.js]: mainly spacing issues and yoda check if possible in code block

**Comment by milindmore22** [comment]
@thrijith addressed feedback on PR please check now

**Pull request merged**